modules/microshift-olm-deploy-ops-spec-ns.adoc

```diff
@@ -92,9 +92,9 @@ spec: <1>
 +
 [source,terminal]
 ----
-$ oc apply -f _<og.yaml>_ <1>
+$ oc apply -f _<og>.yaml_ <1>
```

Comment:
```suggestion
$ oc apply -f _<og.yaml>_ <1>
```

Reply:
Correct as-is.

Reply:
reverting changes to lines 65, 67 and 95